internal/cmd/skupper/site/nonkube/site_status_test.go

```diff
@@ -26,9 +26,12 @@ func TestCmdSiteStatus_ValidateInput(t *testing.T) {
 		expectedError     string
 	}
 
-	tmpDir := filepath.Join(t.TempDir(), "/skupper")
-	err := os.Setenv("SKUPPER_OUTPUT_PATH", tmpDir)
-	assert.Check(t, err == nil)
+	if os.Getuid() == 0 {
+		api.DefaultRootDataHome = t.TempDir()
+	} else {
+		t.Setenv("XDG_DATA_HOME", t.TempDir())
+	}
+	tmpDir := api.GetDataHome()
```

Comment:
I guess these changes have to be replicated in the rest of CLI unit tests, right? (in a subsequent PR)

Reply:
I think, I have already done that:

```
internal/cmd/skupper/link/nonkube/link_generate_test.go
internal/cmd/skupper/site/nonkube/site_update_test.go
internal/cmd/skupper/site/nonkube/site_delete_test.go
internal/cmd/skupper/site/nonkube/site_status_test.go
internal/cmd/skupper/version/nonkube/version_test.go
```

But if something is missing, we can add them later. WDYT?

Reply:
Oh! indeed, I meant that _I_ have to adapt those because I have a pending pull request relying heavily on SKUPPER_OUTPUT_PATH

Reply:
Thank's Noe!